README.md

```diff
@@ -72,96 +73,105 @@ Adverbs are "slowly" and "noisily"
 
 The following are examples of valid sentences:
 
-* The smelly dog ran.
-* The smelly dog ran slowly.
-* The cat ate noisily.
-
-## Q2.1 (7 points)
+The smelly dog ran.
+The smelly dog ran slowly.
+The cat ate noisily.
+Q2.1 (7 points)
 
 Write the BNF (not EBNF) description for this language.
 
-## A2.1
-
-«replace this with your answer»
+A2.1
 
+<sentence> ::= "the" <adjectives> <noun> <verb> <adverb>
+<adjectives> ::= "" | <adjectives> <adjective>
+<adjective> ::= "lazy" | "smelly"
+<noun> ::= "dog" || "cat"
+<verb> ::= "ran" || "ate"
+<adverb> ::= "slowly" | "noisily" | ""
 
-## Q2.2 (5 points)
+Q2.2 (5 points)
 
 Write this grammar using EBNF with common extensions
 
-## A2.2
-
-«replace this with your answer»
-
+A2.2
 
-## Q2.3 (6 points)
+<<<<<<< HEAD
+sentence ::= "the", {<adjective>}, <noun>, <verb>, [<adverb>];
+=======
+sentence ::= "the", {}, , , [];
+>>>>>>> 07f1444fe496e18ef7278164002acd9890749777
```

Comment:
Merge failure. A little tacky, young padawan.

This is good as far as it goes, but  you still need to define the non terminals

# -2